### PR TITLE
improve abstract and introduction

### PR DESCRIPTION
**Pull request by stpeter**
Fixes https://github.com/yaronf/I-D/issues/378

### CODE REVIEW

**Comment by yaronf** [comment]
"Address #xxx" does not trigger the GitHub linkage of PR to issue. Should be "Fixes #xxx".

**Review by yaronf — APPROVED** [review]

File: BCP195bis/draft-ietf-uta-rfc7525bis.md [hunk 1]
```diff
@@ -281,15 +281,15 @@ informative:
...
... variety of application protocols, including HTTP {{HTTP1.1}} {{HTTP2}}, IMAP {{?RFC9051}}, POP {{?STD53}}, SIP {{?RFC3261}}, SMTP {{?RFC5321}}, and XMPP {{?RFC6120}}.  Those protocols use both the TLS handshake protocol and the TLS record layer.  The TLS handshake protocol is also used with different record layers in transport protocols such as Datagram Transport Layer Security (DTLS) and QUIC {{?RFC9000}}.  Over the years leading to 2015, the industry had witnessed serious attacks on the TLS "family" of protocols, including attacks on the most commonly used cipher suites and their modes of operation.  For instance, both the AES-CBC {{?RFC3602}} and RC4 {{!RFC7465}} encryption algorithms, which together were once the most widely deployed ciphers, were attacked in the context of TLS.  Detailed information about the attacks known prior to 2015 is provided in a companion document ({{?RFC7457}}) to the previous version of this specification, which will help the reader understand the rationale behind the recommendations provided here. That document has not been updated in concert with this one; instead, newer attacks are described in this document, as are mitigations for those attacks.
```
Comment (thomas-fossati) [inline BCP195bis/draft-ietf-uta-rfc7525bis.md:292]: It's not entirely accurate to characterise DTLS as reusing the TLS handshake to apply the result to a different record layer. DTLS is armed to address the lack of an underlying reliable pipe abstraction (i.e., amplification potential, retransmission, reordering, path migration, etc.) so even if the bulk of the handshake is the same, there are quite important differences in terms of state machine and messages exchanged.

Comment (stpeter) [reply BCP195bis/draft-ietf-uta-rfc7525bis.md:292]: Hmm, true. I will propose further changes.

Comment (thomas-fossati) [reply BCP195bis/draft-ietf-uta-rfc7525bis.md:292]: QUIC on the other hand behaves exactly as you phrase it: the QUIC transport recreates the TCP service and feeds a reliable, ordered stream into the TLS machinery.

Comment (stpeter) [reply BCP195bis/draft-ietf-uta-rfc7525bis.md]: See https://github.com/yaronf/I-D/pull/390/commits/23824dc685c44103b9b513b5a10f9916c68598ac

**Review by thomas-fossati — APPROVED** [review]
thanks!

**Pull request merged**